app/assets/javascripts/geoblacklight/templates/index_map_info.hbs

```diff
@@ -14,6 +14,15 @@
         <img src="{{thumbnailUrl}}">
       {{/if}}
     {{/if}}
+    {{#if thumbUrl}}
+      {{#if websiteUrl}}
+        <a href="{{websiteUrl}}">
```

Comment:
I was wondering why the thumbnail was not hyperlinked with this new fixture. In the OIM 1.0.0 spec, `digHold` and `websiteUrl` are similar with ambiguous differences, and we've been using digHold primarily. For this fixture, I might include both?

Reply:
I've included both digHold and websiteURL in the OpenIndexMap.